### DISCUSSION TITLE
Hello World 👋 | Mircea Anton

### DISCUSSION THREAD

**Discussion by giscus** [mirceanton/mirceanton.com#7] [Announcements]
# Hello World 👋 · Mircea Anton

My awesome website

https://deploy-preview-24--mirceanton.netlify.app/posts/hello-world/

**Comment by mircea-pavel-anton** [comment]
This is a test to see if comments are working!

**Reply by mircea-pavel-anton** [reply]
It appears they do! 🥳

**Comment by guilsa** [comment]
Hi Mircea! I found you because someone on Reddit linked to your YT video (https://youtu.be/4_U0KK-blXQ). Did you find it better not to self-host your website? What CMS are you running it on and how satisfied are you? Cheers!

**Reply by mircea-pavel-anton** [reply]
Hi! I'm using a static site generator to build my website. It used to be Hugo but nowadays I'm using Jekyll.

As for hosting, it's currently done via GitHub pages. I've been thinking about self hosting it in my cluster but haven't pulled the trigger on that yet. GitHub pages is free and I'm hosting the repository on GitHub as well so that made it super easy.

**Reply by guilsa** [reply]
I see! Thanks for replying. I'm doing some of the KodeKloud course and am contemplating killer.sh and CKA like you mentioned. Your blog was very helpful!

**Reply by mircea-pavel-anton** [reply]
I am happy to hear it was helpful!
Good luck on your certification, though with both KodeKloud and killer.sh, I doubt you'll need any!

**Reply by guilsa** [reply]
I just realized it's a whopping $445. Too much! Just watched some folks practicing on killer.sh. It's pretty specific and requires improving my sed/awk skills, lol. I guess I'm reassessing my goals and objectives. Did the Absolute Beginner KodeKloud course, so let me just focus on the Certified Administrator course for now, and see if I can finish that first!
